gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStoragePathTest.java

```diff
@@ -27,8 +30,10 @@
   @Rule
   public final ExpectedException thrown = ExpectedException.none();
 
```

Comment:
> but why not put the comment on that line?

Because of the github code review tool which only allow to put comments near changed areas.
This is terrible and I am unaware of a way to go around it. Do you?